dev/archery/archery/templates/email_token_expiration.txt.j2

```diff
@@ -0,0 +1,28 @@
+{#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#}
+{%- if True -%}
+{%- endif -%}
```

Comment:
Why do we need this?

Reply:
This was there because the comment block with the license generates an initial empty line that makes the email incorrect:
```
$ CROSSBOW_GITHUB_TOKEN=expired archery crossbow notify-token-expiration --days 40

From: None <None>
To: None
Subject: [CI] Arrow Crossbow Token Expiration in ALREADY_EXPIRED

The Arrow Crossbow Token will expire in 0 days.

Please generate a new Token. Send it to Apache INFRA to update the CROSSBOW_GITHUB_TOKEN.
Update it on the crossbow repository and in the Azure pipelines.
```
but I have pushed a minor change to strip the line on Python instead of Jinja because it was not clear.